README.md

```diff
@@ -34,18 +38,24 @@ Using Gnome Builder is the easiest way to get the app built without even using t
 
 ### Meson
 
-We use TDLib as the backend for the Telegram API, so you'll need to have TDLib already installed in your system (together with GTK4 and libadwaita). Then, you'll need Meson and Rust to actually build the app.
+#### Prerequisites
+
+The following packages are required to build Telegrand:
+
+- meson
+- cargo
+- GTK4
+- libadwaita
```

Comment:
```suggestion
libadwaita (current development version)
```

Reply:
I recall that this has already caused confusion.

Reply:
Right, but `development version` can be also interpreted as the current `alpha` version, so maybe `latest git revision` is more appropriate here.